src/Runtime.Base/src/System/Runtime/TypeCast.cs

```diff
@@ -1027,7 +1027,7 @@ private static unsafe int GetIntegralTypeMatchSize(CorElementType corType)
 
         // source type + target type + assignment variation -> true/false
         [System.Runtime.CompilerServices.EagerStaticClassConstructionAttribute]
-        private static class CastCache
+        internal static class CastCache
```

Comment:
Ah, yes this was needed when I had the eager cctors running after the ordered ones. I'll revert this piece.